Server/Room.cpp

```diff
@@ -95,10 +109,11 @@ void Room::startLobbyLoop(void)
 {
     CommunicatorMessage connectionOperation;
 
+    std::signal(SIGINT, signalCallbackHandler);
     startConnexionProtocol();
     initEcsGameData();
     _state = RoomState::LOBBY;
-    while (_state != RoomState::ENDED && _state != RoomState::UNDEFINED) {
+    while (_state != RoomState::ENDED && _state != RoomState::UNDEFINED && roomState != RoomState::ENDED) {
```

Comment:
Why is there two `_state != RoomState::ENDED` ?

Reply:
Refer to our discussion this afternoon